resources/wikia/modules/localStorage.js

```diff
@@ -5,6 +5,9 @@ define('wikia.localStorage', ['wikia.window'], function(window) {
 	'use strict';
 
 	try {
+		window.localStorage.setItem('localStorageTestItem', 'testValue');
+		window.localStorage.getItem('localStorageTestItem');
```

Comment:
Just wondering: how does this module differ from https://github.com/Wikia/app/blob/b03df0a89ed672697e9c130d529bf1eb25f49cda/resources/wikia/modules/cache.js#L39, which makes basically the same check?